pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+    In a way this section can be thought of as a breakout section of the Rationale section that is focused specifically on why certain ideas were not ultimately pursued.
+
+11. Open Issues - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete and reduces people duplicating prior discussion.
+12. Footnotes - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+13. Copyright/license - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://choosealicense.com/licenses/cc0-1.0/) (see this PHEP for an example).
+
+# PHEP Formats and Templates
+PHEPs are UTF-8 encoded text files using the Markdown format. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
+
+The PHEP text files are automatically rendered by the GitHub interface for easier online reading. The Markdown source is the version of record.
+
+The readability of the diff should be considered when editing a PHEP.
+
+The editors are responsible for enforcing PHEP format and consistency of style within a PHEP. They may use automated tools (e.g. pre-commit hooks) to assist.
+
+# PHEP Header Preamble
+<a name="phep-header-preamble"></a>
+Each PHEP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble, marked off with a [code fence](https://spec.commonmark.org/current/#fenced-code-blocks). The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+      ```
+      PHEP: <phep number>
+      Title: <phep title>
+      Author: <list of authors' real names and optionally, email addrs>
+      Discussions-To: <URL of current canonical discussion thread>
+      Status: <Draft | Accepted | Rejected |
+               Withdrawn | Final | Superseded>
+      Type: <Standards Track | Informational | Process>
+      Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+    * Requires: <phep numbers>
+      Created: <date created on, in dd-mmm-yyyy format>
+      Post-History: <dates, in dd-mmm-yyyy format,
+                     inline-linked to PHEP discussion threads>
+    * Replaces: <phep number>
+    * Replaced-By: <phep number>
+    * Resolution: <url>
+      ```
+
+The Author header lists the names and optionally the email addresses of all the authors/owners of the PHEP. The format of the Author header values must be:
```

Comment:
Just realized this should also include ORCID (at least as a possibility if not required).

Reply:
100% agree, my ORCID is probably the most consistent way to look me up. Let's add at least the possibility to include it (it's possible some people still don't have ORCIDs so I wouldn't _require_ it).

Reply:
Further discussion can go to https://github.com/heliophysicsPy/standards/pull/22/files#r1388544174 , where this change has been made.